<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettings</class>
 <widget class="QDialog" name="AdvancedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Compression</string>
   </property>
   <widget class="QCheckBox" name="cbCompressRLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RLE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCompressHuff">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Huffman</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCompressNone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Encryption</string>
   </property>
   <widget class="QCheckBox" name="cbEncryptXOR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbEncryptNone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Packets</string>
   </property>
   <widget class="QRadioButton" name="rbPacketFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPacketRaw">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bnOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bnCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
